frontend/main.js

```diff
@@ -405,6 +381,31 @@ async function startApp () {
           // Fallback behavior
           this.removeLoadingAnimation()
           alert(L('Error while setting up service worker'))
+          throw e
```

Comment:
Since we're not really signed in at this point, the user doesn't have access to the logs that they can send us.

So could you please update this `alert` message on line 383 to say:

```
alert(L('Error while setting up service worker: {err}', { err: e.message }))
```